src/sass/components/_section--promotion.scss

```diff
@@ -0,0 +1,255 @@
+      }
+
+      .stars {
+        text-align: center;
+
+        a {
+          text-decoration: none;
+          color: $text-color;
+          display: inline-block;
+          font-style: normal;
+          font-variant: normal;
+          text-rendering: auto;
+          line-height: 1;
+          font-family: 'Font Awesome 5 Free';
+          font-weight: 400;
+
+          &.full {
+            font-weight: 900;
+          }
+          &:before {
+            content: "\f005";
+          }
+        }
+      }
+
+      .slider {
+        width: 100%;
+        height: 1px;
+        background: black;
+        margin: 15px 0;
+        position: relative;
+
+        .slide1 {
+          width: 20%;
+          height: 5px;
+          background: $primary;
+          position: absolute;
+          top: 0;
+          left: 40%;
+        }
+      }
+      .price {
+        display: flex;
+        align-items: center;
+        position: absolute;
+        right: 15px;
+        bottom: 0;
+
+        .price-old {
+          font-size: 13px;
+          color: $promotion-dots-color;
+          padding: 5px 10px;
+          text-decoration: line-through $text-color;
+        }
+      }
+    }
+
+    .furniture {
+      max-height: 490px;
+      height: 490px;
+      padding: 0;
+
+      .carousel-inner {
+        position: relative;
+        width: 100%;
+        height: 100%;
+        overflow: hidden;
+      }
+      .furniture1 {
+        background: url(../images/promotion/fur1.jpg);
+      }
+
+      .furniture2 {
+        background: url(../images/promotion/fur2.jpg);
+      }
+
+      .furniture3 {
+        background: url(../images/promotion/fur3.jpg);
+      }
+
+      .carousel-item {
+        height: 100%;
+        margin-top: 0;
+        background-size: cover;
+        background-position: center;
+      }
+
+      .contentBox {
+        padding-top: 145px;
+
+        .content {
+          color: white;
+          text-transform: uppercase;
+          padding: 55px 0 45px 0;
+          background: $counter-time-color-background;
+          position: relative;
+
+          h1, h2 {
+            font-weight: 400;
+
+            span {
+              font-weight: 600;
+            }
+          }
+
+          .btn-main {
+            width: 180px;
+            position: absolute;
+            bottom: -20px;
+            left: 50%;
+            transform: translateX(-90px);
+            border-radius: 3px;
+            border: 1px solid white;
+            background: white;
+            line-height: 30px;
+            color: $text-color;
+            transition: .2s;
+
+            &:hover {
+              background: $text-color;
+            }
+          }
+        }
+      }
+      .buttons {
+        position: relative;
+        background: $text-color;
+        height: 40px;
+        top: -40px;
+
+        .btn-main {
+          height: 40px;
+          border: none;
+          transition: .2s;
+          width: 49.7%;
```

Comment:
Why not `49.8%` or `49.6%` wide? Odd values need to have some comment to explain why you used this particular value. Probably there is another solution to have for example 50% width.